README.md

```diff
@@ -231,15 +231,43 @@ If a given `Animal` has no children, its `child_names` list is empty.
 #### Constraints and Rules
 - `@fold` can only be applied to vertex fields, except the root vertex field.
 - May not exist at the same vertex field as `@recurse`, `@optional`, or `@output_source`.
-- Expanding vertex fields is not allowed within a scope marked `@fold`.
-- `@tag` and `@fold` may not be used within a scope marked `@fold`.
+- At most one vertex field may be expanded in a scope within a `@fold` marked scope. Note that this applies for all nested scopes (see example below).
```

Comment:
This is a bit unclear -- how about moving around the word order:
`In the scope marked `@fold` and any scope nested within, at most one vertex field may be expanded. (Example below)`

I think this will make the "Note that" sentence unnecessary.